docs/changelog-configuration.md

```diff
@@ -103,6 +103,13 @@ YAML encoding that is compatible with ansible-lint's default rules.
     The exact format used might be adjusted in the future if new releases of ansible-lint
     adjust their yamllint configuration.
 
+### `changelog_semantic_versioning_sort` (boolean)
```

Comment:
Since the changelog tool can handle also different versions than semantic versions (for example for ansible-core and other packages), I think it would be better to adjust the name of this option, maybe:
```suggestion
### `changelog_version_sort` (boolean)
```